sycl/doc/GetStartedGuide.md

```diff
@@ -21,40 +21,40 @@ and a wide range of compute accelerators such as GPU and FPGA.
 
 ## Prerequisites
 
-* `git` - https://git-scm.com/downloads
-* `cmake` version 3.2 or later - http://www.cmake.org/download/
-* `python` - https://www.python.org/downloads/release/python-2716/
-* `ninja` - https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages
+* `git` - [Download](https://git-scm.com/downloads)
+* `cmake` version 3.2 or later - [Download](http://www.cmake.org/download/)
+* `python` - [Download](https://www.python.org/downloads/release/python-2716/)
+* `ninja` -
+[Download](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
 * C++ compiler
   * Linux: `GCC` version 5.1.0 or later (including libstdc++) -
-    https://gcc.gnu.org/install/
+    [Download](https://gcc.gnu.org/install/)
   * Windows: `Visual Studio` version 15.7 preview 4 or later -
-    https://visualstudio.microsoft.com/downloads/
+    [Download](https://visualstudio.microsoft.com/downloads/)
 
 ### Create DPC++ workspace
 
 Throughout this document `DPCPP_HOME` denotes the path to the local directory
 created as DPC++ workspace. It might be useful to create an environment variable
 with the same name.
 
-**Linux**
+**Linux**:
```

Comment:
do we need semicolomn?

Reply:
Yep. Otherwise linter recommends replacing it with heading.